sdk/tables/azure-data-tables/inc/azure/data/tables/tables_clients.hpp

```diff
@@ -307,7 +307,7 @@ namespace Azure { namespace Data { namespace Tables {
      * @param partitionKey The partition key of the entity.
      * @param rowKey The row key of the entity.
      * @param context for canceling long running operations.
-     * @return Entity list paged response.
+     * @return Table entity.
```

Comment:
Is this the type of documentation that's available in the REST API spec?

It's essentially redundant and doesn't provide much helpful info, given it just reiterates what's already visible in the function signature :(

Reply:
the comment overall not just the return object explains the method. it returns a table entity, thus the return comment, indeed taken out of context does not say much, but it is correct.

Reply:
That's fair.
Although, it is a generic return type in .NET, with the type parameter documented a bit more (as an interface):
https://learn.microsoft.com/en-us/dotnet/api/azure.data.tables.tableclient.getentity?view=azure-dotnet#type-parameters
https://learn.microsoft.com/en-us/dotnet/api/azure.data.tables.itableentity?view=azure-dotnet

The main issue in our docs is that even the returned `TableEntity` model type is documented very generically.

Compare this to Java's, where "the entity" has an explanation:
https://github.com/Azure/azure-sdk-for-cpp/blob/8973c6c9fbf025eb05ca41ce645cff3c2566a980/sdk/tables/azure-data-tables/inc/azure/data/tables/models.hpp#L444-L472

https://azuresdkartifacts.blob.core.windows.net/azure-sdk-for-java/staging/apidocs/com/azure/data/tables/TableClient.html#getEntity(java.lang.String,java.lang.String)

https://azuresdkartifacts.blob.core.windows.net/azure-sdk-for-java/staging/apidocs/com/azure/data/tables/models/TableEntity.html